main.py

```diff
@@ -1,31 +1,12 @@
 import argparse
+from converter.converter import *
 
 
 CELSIUM_INPUT = ['celsius', 'cels', 'cel', '-c', 'c']
 FAH_INPUT = ['fahrenheit', 'fahren', 'far', '-f', 'f']
 QUIT_INPUT = ['quit', 'q']
```

Comment:
@MariaMaximova, I have suggession to add `exit` here, just in case

Reply:
Added in [main.py](https://github.com/MariaMaximova/convert_temp/commit/67f37cda14e8921dad8a4e3e96f9f8abb4ad93d8)